test/functional/test_runner.py

```diff
@@ -455,6 +459,14 @@ def __init__(self, name, status, time):
         self.time = time
         self.padding = 0
 
+    def sort_key(self):
+        if self.status == "Passed":
+            return 1, self.name.lower()
+        elif self.status == "Failed":
+            return 2, self.name.lower()
+        elif self.status == "Skipped":
+            return 0, self.name.lower()
```

Comment:
My preference would be to place skipped tests below passed tests for visibility, so the order would be:

<passed tests>
<skipped tests>
<failed tests>
SUMMARY

Reply:
Fair point, I'll swap the passed and skipped